def err(errcode):
  print("There was an error: " + errcode)
